mimic/canned_responses/fastly.py

```diff
@@ -0,0 +1,278 @@
+"""
+Canned response for fastly
+"""
+import uuid
+
+
+class FastlyResponse(object):
+
+    """
+    Canned response for fastly.
+    See docs here https://docs.fastly.com/api/config
+    """
+
+    fastly_cache = {}
+
+    def get_current_customer(self):
+        """
+        Returns the current customer with response code 200.
+
+        :return: a JSON-serializable dictionary matching the format of the JSON
+                 response for fastly_client.get_current_customer()
+                 ("/current_customer") request.
+        """
+        current_customer = {
+            u'can_edit_matches': u'0',
+            u'can_read_public_ip_list': u'0',
+            u'can_upload_vcl': u'1',
+            u'updated_at': u'2014-11-03T23:37:44+00:00',
+            u'has_config_panel': u'1',
+            u'has_improved_ssl_config': False,
+            u'id': u'3bKRHISECARTtoUMxW6',
+            u'has_historical_stats': u'1',
+            u'has_openstack_logging': u'0',
+            u'can_configure_wordpress': u'0',
+            u'has_improved_logging': u'1',
+            u'readonly': '',
+            u'ip_whitelist': u'0.0.0.0/0',
+            u'owner_id': u'asewQWEwdRnLnCKyMztZqkQvy',
```

Comment:
This looks … suspiciously specific, as does the `'id'` above.  I think it might be a good idea to generate some random IDs here?  Normally I would say this could follow on in a separate change, but if we don't do that in the first revision where this is added, then tests may start to depend on these hard-coded IDs.